### PR TITLE
Adding an instance_status variable which gets emitted with instance e…

### PR DESCRIPTION
**Pull request by bkpeterson**
Added a variable to keep track of the status of each instance, which also gets emitted with instance_errors in calculateErrors.  This will be used in the bitfocus-companion module to provide instance-specific status feedback.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "companion",
-	"version": "2.2.0",
```
Comment (Julusian) [inline package.json]: Please don't change the version of core in this pr

**Review by Julusian — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (bkpeterson) [reply package.json]: Sorry, reverted.

**Comment by bkpeterson** [comment]
I am waiting on this for an install - any thoughts from the reviewers?  Any further changes or can this be merged?

**Comment by krocheck** [comment]
I have implemented this an alternate way.  There was no need to add `self.instance_status` since `self.status` already exists with the same array.

**Pull request closed without merging**